lib/mimdb_web/views/actor_view.ex

```diff
@@ -0,0 +1,7 @@
+defmodule MimdbWeb.ActorView do
+  use MimdbWeb, :view
+
+  def age(date_fn, actor) do
```

Comment:
I know this was my idea and it is awesome you worked through it! So take this comment as a commented directed at me and not you....ask yourself, can you get away with just passing the date or does passing a function that generates the date open up more powerful things that you need? An interesting thing to ponder that may give you added clarity around when you should use simple parameters and when you must escalate to higher order function.

Reply:
Thank you @mratliff, this is a good tip to have in mind